subgraphs/cryptopunk/src/constants/initializers.ts

```diff
@@ -0,0 +1,273 @@
+import { Address, BigInt, ethereum } from "@graphprotocol/graph-ts";
+import {
+  Collection,
+  CollectionDailySnapshot,
+  Marketplace,
+  MarketplaceDailySnapshot,
+  Trade,
+  _User,
+  _Bid,
+} from "../../generated/schema";
+import * as constants from "./constants";
+
+export function getOrCreateMarketplace(): Marketplace {
+  let marketplace = Marketplace.load(
+    constants.CRYPTOPUNK_MARKET_CONTRACT_ADDRESS
+  );
+  if (!marketplace) {
+    marketplace = new Marketplace(constants.CRYPTOPUNK_MARKET_CONTRACT_ADDRESS);
+    marketplace.name = constants.MARKETPLACE_NAME;
+    marketplace.slug = constants.MARKETPLACE_SLUG;
+    marketplace.network = constants.Network.MAINNET;
+    marketplace.schemaVersion = "1.0.0";
+    marketplace.subgraphVersion = "1.0.0";
+    marketplace.methodologyVersion = "1.0.0";
+    marketplace.collectionCount = 1;
+    marketplace.tradeCount = 0;
+    marketplace.cumulativeTradeVolumeETH = constants.BIGDECIMAL_ZERO;
+    marketplace.marketplaceRevenueETH = constants.BIGDECIMAL_ZERO;
+    marketplace.creatorRevenueETH = constants.BIGDECIMAL_ZERO;
+    marketplace.totalRevenueETH = constants.BIGDECIMAL_ZERO;
+    marketplace.cumulativeUniqueTraders = 0;
+    marketplace.save();
+  }
+  return marketplace;
+}
+
+export function getOrCreateCollection(): Collection {
+  let collection = Collection.load(
+    constants.CRYPTOPUNK_COLLECTION_CONTRACT_ADDRESS
+  );
+  if (!collection) {
+    collection = new Collection(
+      constants.CRYPTOPUNK_COLLECTION_CONTRACT_ADDRESS
+    );
+    collection.name = constants.CRYPTOPUNK_NAME;
+    collection.symbol = constants.CRYPTOPUNK_SYMBOL;
```

Comment:
I think the symbol for cryptopunk is PUNK

Reply:
I took the symbol from the contract initializer. 
<img width="808" alt="Screenshot 2022-09-08 at 8 04 07 AM" src="https://user-images.githubusercontent.com/46936675/189020729-6ac4cf44-e4c1-4953-864e-a61ed5a7d1c0.png">
Should I change it to PUNK?